src/kt_method.h

```diff
@@ -0,0 +1,25 @@
+#ifndef GODOT_JVM_KT_METHOD_H
+#define GODOT_JVM_KT_METHOD_H
+
+
+#include <core/object.h>
+#include "kt_member.h"
+
+class KtMethod : KtMember {

```

Comment:
This should be named `KtFunction` to match the jvm side. Additionally this should wrap the jvm `KtFunction` so it should extend `JavaInstanceWrapper`. Also the info needed should be fetched in the constructor, i.e method name, number parameters etc..